metrics/web/lib/common/presentation/scaffold/widget/metrics_scaffold.dart

```diff
@@ -22,18 +27,43 @@ class MetricsScaffold extends StatelessWidget {
     Key key,
     @required this.body,
     this.drawer = const MetricsDrawer(),
-    this.padding = const EdgeInsets.symmetric(horizontal: 124.0),
+    this.padding = EdgeInsets.zero,
+    this.bodyTitle,
```

Comment:
```suggestion
    this.title,
```
or `pageTitle` (as we use it in the MetricsPageTitle)

Also, would it make sense to put this named parameter to the beginning of the parameters list making the order of reading more understandable?